packages/@react-aria/table/test/tableResizingTests.tsx

```diff
@@ -173,8 +173,8 @@ export let resizingTests = (render, rerender, Table, ControlledTable, resizeCol,
       it.each`
       col         | delta  | expected                     | expectedOnResize
       ${'Name'}   | ${-50} | ${[75, 103, 103, 103, 516]}  | ${[75, '1fr', '1fr', '1fr', '5fr']}
-      ${'Name'}   | ${50}  | ${[150, 94, 94, 94, 468]}    | ${[150, '1fr', '1fr', '1fr', '5fr']}
-      ${'Type'}   | ${-50} | ${[100, 75, 104, 104, 517]}  | ${[100, 75, '1fr', '1fr', '5fr']}
+      ${'Name'}   | ${50}  | ${[150, 94, 94, 93, 469]}    | ${[150, '1fr', '1fr', '1fr', '5fr']}
```

Comment:
you'll see a pixel shift around here and there in the tests due to the new algorithm, the important thing is that the sum was maintained
we could change how we redistribute the remainders if we want, this was just an easy way to do it